omod/src/main/webapp/resources/htmlforms/015-HMIS_ACP_ExposedInfantClinicalChart-EncounterPage.xml

```diff
@@ -0,0 +1,422 @@
+                                <obs  conceptId="1193" answerConceptId="244" answerLabel="Mebendazole"/>
+                            </div>
+                            <div class="col-md-3">
+                                <obs  conceptId="1193" answerConceptId="237"
+                                     answerLabel="Metronidazole"/>
+                                <obs  conceptId="1193" answerConceptId="461"
+                                     answerLabel="Multivitamins"/>
+                                <obs  conceptId="1193" answerConceptId="99054"
+                                     answerLabel="Nutrition Support"/>
+                                <obs  conceptId="1193" answerConceptId="919" answerLabel="Nystatin"/>
+                                <obs  conceptId="1193" answerConceptId="89" answerLabel="Paracetamol"/>
+                                <obs conceptId="1193" answerConceptId="1194" answerLabel="RH"/>
+                                <obs conceptId="1193" answerConceptId="99127" answerLabel="RHE"/>
+                            </div>
+                            <div class="col-md-3">
+                                <obs conceptId="1193" answerConceptId="768" answerLabel="RHZ"/>
+                                <obs  conceptId="1193" answerConceptId="1131" answerLabel="RHZE"/>
+                                <obs conceptId="1193" answerConceptId="99128" answerLabel="RHZES"/>
+                                <obs  conceptId="1193" answerConceptId="916" answerLabel="Septrin"/>
+                                <obs  conceptId="1193" answerConceptId="329" answerLabel="Vit B Complex"/>
+                                    <span id="medication">
+                                        <span id="trigger">
+                                            <obs  conceptId="1193" answerConceptId="90002" answerLabel="Other specify:"/>
+                                        </span>
+                                        <span id="disabled">
+                                            <obs  conceptId="99035" answerClasses="Drug" style="autocomplete"/>
+                                        </span>
+                                    </span>
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-md-8">
+                                <label>ACTION(S) TAKEN if any care indicator are a concern</label>
+                                <obs conceptId="162865" style="textarea"/>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-md-10"></div>
+                    <div class="col-md-2">
+                        <span class="float-right">
+                            <submit submitStyle="customSubmit" class="confirm" submitLabel="Save"/> 
```

Comment:
@semujju All submit buttons are meant to be at the top

Reply:
That is true for forms with tabs

Reply:
This form has no tabs, it better to have the button down